flink-table/flink-table-planner/src/test/scala/org/apache/flink/table/runtime/batch/table/CalcITCase.scala

```diff
@@ -593,6 +593,77 @@ class CalcITCase(
     val results = filterDs.toDataSet[Row].collect()
     TestBaseUtils.compareResultAsText(results.asJava, expected)
   }
+
+  @Test
+  def testAddColumns(): Unit = {
```

Comment:
Does it make sense to have those IT cases? After all it just produces a `Projection`, which is already covered by tests.

Reply:
In general, it's better for each operator to add an ITCase.  and I know it will cost more time, do you want to remove all of the ITCase, or reduce the column ITCase in one?.